lib/iex/test/iex/helpers_test.exs

```diff
@@ -1347,6 +1347,7 @@ defmodule IEx.HelpersTest do
     test "builds a PID from string" do
       assert inspect(pid("0.32767.3276")) == "#PID<0.32767.3276>"
       assert inspect(pid("0.5.6")) == "#PID<0.5.6>"
+      assert inspect(pid(:init) == "#PID<0.0.0>")
```

Comment:
It looks like `inspect` should be around `pid/1` function, currently this test will never fail

Reply:
Good catch, please send a PR?